src/main/java/net/minecraftforge/client/model/ItemTextureQuadConverter.java

```diff
@@ -44,7 +44,7 @@ private ItemTextureQuadConverter()
      * Takes a texture and converts it into BakedQuads.
      * The conversion is done by scanning the texture horizontally and vertically and creating "strips" of the texture.
      * Strips that are of the same size and follow each other are converted into one bigger quad.
-     * </br>
+     * <br/>
```

Comment:
Sure, though I still believe this should be stripped and done using `<p>`, though that's an out of scope comment for this PR.